docs/maya/skin.md

```diff
@@ -6,16 +6,19 @@ The influence these constraints have on the simulated mesh can be freely modifie
 
 ### How To Use
 
-The AdnSkin deformer is of great simplicity to set up and apply to a mesh within a Maya scene. The way this deformer works is by applying simulation on top of the skin mesh (simulated mesh) which will be directly coupled to its reference mesh (with deformation over time).
+The AdnSkin deformer is of great simplicity to set up and apply to a mesh within a Maya scene. The way this deformer works is by applying simulation on top of the skin mesh (simulated mesh) which will be directly coupled to its target meshes (with deformation over time).
 
 To create an AdnSkin deformer within a Maya scene, the following inputs must be provided:
 
-  - **Reference Mesh (R)**: Mesh to drive the simulation skin (e.g. fascia or combined muscles).
+  - **Targets (T)**: List of geometries to drive the simulation mesh (e.g. select the muscles to simulate the fascia, select the fascia to simulate the fat, select the fat to simulate the skin).
   - **Skin Mesh (S)**: Mesh to apply the deformer onto.
 
+> [!NOTE]
+> It is not mandatory to select the targets on creation of the AdnSkin deformer, Targets can be added and removed after creating the deformer. For more information check the advanced [section](#targets).
+
 The process to create an AdnSkin deformer is:
 
-1. Select the **Reference Mesh**, then the **Skin Mesh**.
+1. Select the **Targets** (or add them later), then the **Skin Mesh**.
```

Comment:
```suggestion
1. Select the **Targets** (optional, they can be added later), then the **Skin Mesh**.
```

Reply:
Done. Added this suggestion in "A Simple Setup" page where I also wrote "or add them later"